## <i> <span style='color:red'>HYPOTHESIS TESTING:BUYER RATIO</i></span>
### PROBLEM STATMENT:Sales of products in four different regions is tabulated for males and females. Find if male-female buyer rations are similar across regions.

### Import libraries

In [1]:
import pandas as pd
import scipy
from scipy.stats import chi2_contingency

### Read the csv file

In [2]:
data=pd.read_csv("BuyerRatio.csv")
data.columns=['ObsVal','E','W','N','S']
data

ObsVal    E     W     N    S
0    Males   50   142   131   70
1  Females  435  1523  1356  750

### H0 - all proportions of male-female buyer ratio is same
### H1 - all proportions of male-female buyer ratio is not the same

In [3]:
data.drop('ObsVal',axis=1,inplace=True)

### Using chisquare test

In [4]:
scipy.stats.chi2_contingency(data)

(1.595945538661058,
 0.6603094907091882,
 3,
 array([[  42.76531299,  146.81287862,  131.11756787,   72.30424052],
        [ 442.23468701, 1518.18712138, 1355.88243213,  747.69575948]]))

## Inference:
### P value is 0.66>0.05. so, we accept H0. All proportions of male-female buyer ratio is same for all direction